config/config.go

```diff
@@ -26,6 +26,38 @@ func NewConfig() Configuration {
 }
 
 func setConfigDefaults(v *viper.Viper) {
+	v.SetDefault("port", 2424)
+	v.SetDefault("admin_port", 2525)
+	v.SetDefault("log.level", "info")
+	v.SetDefault("backend.type", "memory")
+	v.SetDefault("backend.aerospike.host", "")
+	v.SetDefault("backend.aerospike.port", 0)
+	v.SetDefault("backend.aerospike.namespace", "")
+	v.SetDefault("backend.aerospike.default_ttl_seconds", 0)
+	v.SetDefault("backend.azure.account", "")
+	v.SetDefault("backend.azure.key", "")
+	v.SetDefault("backend.cassandra.hosts", "")
+	v.SetDefault("backend.cassandra.keyspace", "")
+	v.SetDefault("backend.memcache.hosts", []string{""})
+	v.SetDefault("backend.redis.host", "")
+	v.SetDefault("backend.redis.port", 0)
+	v.SetDefault("backend.redis.password", "")
+	v.SetDefault("backend.redis.db", 0)
+	v.SetDefault("backend.redis.expiration", 0)
+	v.SetDefault("backend.redis.tls.enabled", false)
+	v.SetDefault("backend.redis.tls.insecure_skip_verify", false)
+	v.SetDefault("compression.type", "")
+	v.SetDefault("metrics.type", "")
```

Comment:
I vote to omit `metrics.type` since we're moving to deprecate it.

Reply:
But if we remove it from here and it also gets removed form config.yaml for some reason, prebid-cache implementations that still use this feature won't be able to read the `PBC_METRICS_CONFIG` environment variable even if they set it correctly. Should we keep it just to be safe?

Reply:
Nah. It's deprecated. We only left it in for backwards compatibility. I don't think that applies here since it's new functionality.

Reply:
Done